pennylane_qrack/qrack_device.py

```diff
@@ -647,20 +657,36 @@ def expval(self, observable, **kwargs):
         # estimate the ev
         return np.mean(self.sample(observable))
 
+
+    def _generate_sample(self):
+        rev_sample = self._state.m_all()
+        sample = 0
+        for i in range(self.num_wires):
+            if (rev_sample & (1 << i)) > 0:
```

Comment:
[nit]: Unnecessary parens
```suggestion
            if rev_sample & (1 << i) > 0:
```

Reply:
This is something I learned from C++ Qrack, and it became part of our standards: Do 90% of developers or more _truly_ know off the top of their head that bitwise-AND has higher operator precedence than "greater than"? Does it _hurt_ anything to leave the explicit indication of order-of-operations? I'd just leave them.